frontend/www/app/settings/personal.html

```diff
@@ -22,6 +22,17 @@
       <span class="input-label" translate>Nickname</span>
       <input type="text" ng-model="currentUser.profile.nickname" placeholder={{"Select_your_nickname"|translate}} ng-change="setPersonalProfileChanged()">
     </label>
+     <label class="item item-input" >
```

Comment:
These two fields should not be seen for Stockholm users.

Check out household.html to see how things can be hidden for Stockholm case.